webots_ros2_driver/scripts/webots_tcp_client.py

```diff
@@ -35,7 +37,8 @@ def host_shared_folder():
 while tcp_socket.connect_ex((HOST, PORT)) != 0:
     print('WARNING: Unable to start Webots. Please start the local simulation server on your host machine. Next connection attempt in 1 second.', file=sys.stderr)
     time.sleep(1)
-tcp_socket.sendall(host_shared_folder().encode('utf-8'))
+command = 'webots ' + os.path.join(host_shared_folder(), world_name) + ' ' + launch_arguments
```

Comment:
So we are not passing the full path to the webots binary here?
This assume the local server script should prepend the WEBOTS_HOME to it?
Do we want this?

Reply:
As discussed, here we only send 'webots' as executable. The server is responsible to append the WEBOTS_HOME prefix.